Header header

uint32 sentinel
uint32 version

Frame frame

Pixel pixel

Line line

Camera camera

FPA fpa

Thermography thermography
